README.md

```diff
@@ -15,7 +15,10 @@ nvm use
 # install your dependencies
 npm i
 
-# more commands to come with react + es6 transpilation
+# start watching the file system for changes
+npm run dev
+
+# TODO: hot reload chrome extension code to make dev loop faster
```

Comment:
Made an issue for this as well https://github.com/mazzola/dnd-beyond-dice-chrome-extension/issues/12